xml/System.Dynamic/DynamicMetaObject.xml

```diff
@@ -26,12 +26,14 @@
   <Docs>
     <summary>Represents the dynamic binding and a binding logic of an object participating in the dynamic binding.</summary>
     <remarks>
-      <format type="text/markdown"><![CDATA[  
+      <format type="text/markdown">
+        <![CDATA[  
   
 ## Remarks  
- For more information, see documentation on the [CodePlex](http://go.microsoft.com/fwlink/?LinkId=141028) Web site and the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).  
+ For more information, see the MSDN article [Implementing Dynamic Interfaces](http://go.microsoft.com/fwlink/?LinkID=198161).  
```

Comment:
For more information, download \[Implementing Dynamic Interfaces](http://download.microsoft.com/download/5/4/B/54B83DFE-D7AA-4155-9687-B0CF58FF65D7/implementing-dynamic-interfaces.pdf) in PDF format.